cardform/src/main/java/com/mercadolibre/android/cardform/presentation/ui/CardFormFragment.kt

```diff
@@ -304,12 +325,14 @@ internal class CardFormFragment : RootFragment<InputFormViewModel>() {
     private fun returnResult(data: CardResultDto, resultCode: Int) {
         activity?.apply {
             if (fromFragment) {
-                supportFragmentManager.popBackStackImmediate()
-                getCurrentFragment(supportFragmentManager)?.onActivityResult(
-                    requestCode,
-                    resultCode,
-                    buildResultIntent(data)
-                )
+                postDelayed((1000L)) {
+                    getCurrentFragment(supportFragmentManager)?.onActivityResult(
+                        requestCode,
+                        resultCode,
+                        buildResultIntent(data)
+                    )
+                    supportFragmentManager.popBackStackImmediate()
```

Comment:
@DIbarraML why are we changing the order of the calls?

`supportFragmentManager.popBackStackImmediate()` was initially called before `getCurrentFragment(...)`

Reply:
sorry, I was trying to find another solution and forgot to undo this change